graphics/oofcanvas/Portfile

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+PortGroup           active_variants 1.1
+
+name                oofcanvas
+version             1.0.1
+revision            0
+
+license             public-domain
+categories          graphics
+platforms           darwin
+
+maintainers         {@snarkhunter nist.gov:stephen.langer} \
+                    nist.gov:oof_manager
+
+description         C++ library that does some of what libgnomecanvas used to do
+
+long_description    OOFCanvas is a replacement for libgnomecanvas, designed \
+                    for use in OOF2, but hopefully useful elsewhere. OOFCanvas \
+                    is based on Cairo and, unlike libgnomecanvas, is compatible\
+                    with gtk3. The canvas is a drawing area that can display a \
+                    variety of shapes, including text. It can be scrolled, \
+                    zoomed, and printed.  Items drawn on the canvas can be \
+                    selected with the mouse. Note that OOFCanvas is not a \
+                    drop-in replacement for libgnomecanvas.  It's also not a \
+                    full-fledged gtk widget.  It's a set of classes that does \
+                    some of what libgnomecanvas did and uses gtk.
+
+homepage            http://www.ctcms.nist.gov/oof/oofcanvas
+master_sites        http://www.ctcms.nist.gov/oof/oofcanvas/source
+distfiles           oofcanvas-${version}
+
+checksums           rmd160 1868451056ad15636f1a03cc165e0296ca58bf28 \
+                    sha256 15760ce9c8bda341e6160e55899ed6bc8056a5fd1f9b1fb68f64fe9c0a3700b4 \
+                    size 1111805
+
+use_configure       no
+python.default_version  27
```

Comment:
can this use a modern Python version please; we preferable do not want to add EOL versions anymore...